### PR TITLE
feat: feed data to recording page ui

### PR DESCRIPTION
**Pull request by BusraTemizyurek**

### CODE REVIEW

File: src/main-page.ts [hunk 1]
```diff
@@ -89,6 +87,7 @@ class MainPage implements Page {
 
             this._calibrationModal?.hide();
             this._recordButton?.classList.remove("d-none");
+
```
Comment (altinokdarici) [inline src/main-page.ts:90]: nit: unnecessary new line.

File: src/recording-page.ts [hunk 2]
```diff
@@ -1,8 +1,25 @@
+interface Point {
```
Comment (altinokdarici) [inline src/recording-page.ts:1]: is this point interface different than the one in the types.ts? Either rename them differently or remove one of them.

Comment (BusraTemizyurek) [reply src/recording-page.ts]: I removed this one

File: src/main-page.ts (hunk 1 above)
Comment (BusraTemizyurek) [reply src/main-page.ts]: deleted

**Review by altinokdarici — APPROVED** [review]

**Pull request merged**